corehq/messaging/scheduling/forms.py

```diff
@@ -1757,6 +1770,10 @@ def create_form_helper():
     def form_choices(self):
         return [(form['code'], form['name']) for form in get_form_list(self.domain)]
 
+    @property
+    def can_use_connect(self):
```

Comment:
Nitpick: is this better than just having the calling code check the feature flag?